packages/meditrak-app/app/database/types/PermissionGroup.jsx

```diff
@@ -13,13 +13,12 @@ PermissionGroup.schema = {
   properties: {
     id: 'string',
     name: { type: 'string', default: 'PermissionGroup not properly synchronised' },
+    parentId: { type: 'string', optional: true },
   },
 };
 
 PermissionGroup.requiredData = ['name'];
 
 PermissionGroup.construct = (database, data) => {
-  const { parentId, ...restOfData } = data;
-  const permissionGroupObject = restOfData;
-  return database.update('PermissionGroup', permissionGroupObject);
+  return database.update('PermissionGroup', data);
```

Comment:
Added `parentId` back here so I can get ancestors of permission groups for the user question

Reply:
Ah if we're making this change, we might need to do a full resync of `permission_groups`?